lib/util.js

```diff
@@ -191,59 +192,47 @@ function getLink(obj, linkName, altName) {
   } else {
     return link;
   }
-}
+};
+
+util.getNativeConsole = function() {
+  return window.console;
+};
 
-function warn(text) {
+util.getConsole = function() {
+  var nativeConsole = util.getNativeConsole();
```

Comment:
Tested with @nikhilvenkatraman-okta. In the tested app, console doesn't exist, so these changes work.